core/src/main/java/nl/nn/adapterframework/jdbc/DirectQuerySender.java

```diff
@@ -46,9 +46,7 @@ public void configure() throws ConfigurationException {
 	public void configure(boolean trust) throws ConfigurationException {
 		super.configure();
 		if (!trust) {
-			ConfigurationWarnings configWarnings = ConfigurationWarnings.getInstance();
-			String msg = "The class ["+getClass().getName()+"] is used one or more times. Please change to ["+FixedQuerySender.class.getName()+"] for better security";
-			configWarnings.add(log, msg, true);
+			ConfigurationWarnings.add(log, "The class ["+getClass().getName()+"] is used one or more times. Please change to ["+FixedQuerySender.class.getName()+"] for better security");
```

Comment:
Deze melding moeten we toch ook wat aan doen.
Minimaal wijzen op het gevaar van SQL injection.